### PR TITLE
Results page

### PR DESCRIPTION
**Pull request by Maliek**
Results page creation + redirection

### CODE REVIEW

File: src/main/resources/js/bundle.js [hunk 1]
```diff
@@ -0,0 +1 @@
+!function(e,t){"object"==typeof exports&&"undefined"!=typeof module?t():"function"==typeof define&&define.amd?define(t):t()}(this,function(){"use strict";new d3pie("results",{data:{content:options}})});
```
Comment (Haroenv) [inline src/main/resources/js/bundle.js]: remove this file, it's changed in #56

**Comment by Haroenv** [comment]
Looks pretty good

```
com.pollistics.controllers.PollControllerTests > voteTest FAILED
    java.lang.AssertionError: Redirected URL expected:</5830364e1c27ea512eea301a> but was:</5830364e1c27ea512eea301a/results>
        at org.springframework.test.util.AssertionErrors.fail(AssertionErrors.java:54)
        at org.springframework.test.util.AssertionErrors.assertEquals(AssertionErrors.java:81)
        at org.springframework.test.web.servlet.result.MockMvcResultMatchers$3.match(MockMvcResultMatchers.java:124)
        at org.springframework.test.web.servlet.MockMvc$1.andExpect(MockMvc.java:171)
        at com.pollistics.controllers.PollControllerTests.voteTest(PollControllerTests.java:178)
```

change the redirection that was expected in the unit test though

File: src/main/resources/templates/polls/results.twig [hunk 2]
```diff
@@ -26,4 +33,6 @@
 			}
 		});
 	</script>
+
+	<script src="/js/dist/vote.js"></script>
```
Comment (Haroenv) [inline src/main/resources/templates/polls/results.twig]: This one should be `result.js`, AFAIK I already made it, if I didn't then you should copy it from vote what's necessary there (maybe some things can be removed then)

Comment (Maliek) [reply src/main/resources/templates/polls/results.twig]: My bad 😞

**Review by Haroenv — APPROVED** [review]
pretty great LGTM

**Pull request merged**